autogluon/utils/tabular/ml/trainer/abstract_trainer.py

```diff
@@ -1782,14 +1785,13 @@ def distill(self, X_train=None, y_train=None, X_val=None, y_val=None,
             if self.problem_type != REGRESSION:
                 self.regress_preds_asprobas = True
 
-        self.time_train_start = time.time()
```

Comment:
safe to remove because the private variables only matter for multi-layer model training, and are not needed for single-layer model training (which distilled models are)